strings.go

```diff
@@ -0,0 +1,368 @@
+package main
+
+import (
+	"go/ast"
+	"go/token"
+	"log"
+	"strconv"
+	"strings"
+
+	"golang.org/x/tools/go/ast/astutil"
+)
+
+func obfuscateStrings(files []*ast.File) []*ast.File {
+
+	rmConst := func(cursor *astutil.Cursor) bool {
+		node := cursor.Node()
+
+		t, ok := node.(*ast.GenDecl)
+		if !ok {
+			return true
+		}
+
+		if t.Tok == token.CONST {
+			t.Tok = token.VAR
+		}
```

Comment:
document this - that is, when obfuscating literals, constants will generally not be possible since the values are computed at init time.